.github/pull_request.yml

```diff
@@ -0,0 +1,34 @@
+name: Pull Request
+on: [pull_request, push]
+
+jobs:
+  make_test:
```

Comment:
super nit: You might want to change the naming of this workflow/job since they end up in the format `{workflow} / {job} (pull_request)` on PRs